api/src/main/java/org/openmrs/module/fhir2/api/translators/OrderIdentifierTranslator.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators;
+
+import org.hl7.fhir.r4.model.Identifier;
+import org.openmrs.Order;
+
+public interface OrderIdentifierTranslator {
```

Comment:
This should really `extend ToFhirTranslator<Order>` or, better yet:

```java
public interface OrderIdentifierTranslator<T extends Order> extends OpenmrsFhirTranslator<T, Identifier>
```

Reply:
I'd go with: 
```java
public interface OrderIdentifierTranslator extends OpenmrsFhirTranslator<Order, Identifier>
```